sw/device/lib/dif/dif_uart.h

```diff
@@ -133,7 +133,7 @@ extern const uint32_t kDifUartFifoSizeBytes;
  */
 OT_WARN_UNUSED_RESULT
 dif_result_t dif_uart_configure(const dif_uart_t *uart,
-                                dif_uart_config_t config);
+                                dif_uart_config_t config, dif_toggle_t enabled);
```

Comment:
nit: Missing API doc for the new parameter. What does it do?

Also, I'd recommend splitting RX and TX into separate enables. The data paths are independent, right?

The enable could probably be part of the config struct as well. :thinking: Up to you.

Reply:
Oh right, good catch.

Sure I can split up the enables, they are separate data paths I believe. 

Let me handle these in a followup PR if that is OK? to make it  easier to review.

Reply:
fixed in #17046